cmip6_downscaling/workflows/bcsd_flow.py

```diff
@@ -96,9 +88,27 @@
     tags=['dask-resource:TASKSLOTS=1'],
     log_stdout=True,
     result=XpersistResult(results_cache_store, serializer="xarray.zarr"),
-    target="postprocess-results-" + target_naming_str,
+    target="postprocess-results-" + target_naming_str + ".zarr",
+)
+
+monthly_summary_task = task(
+    monthly_summary,
+    tags=['dask-resource:TASKSLOTS=1'],
+    log_stdout=True,
+    result=XpersistResult(results_cache_store, serializer="xarray.zarr"),
+    target="monthly-summary-"
+    + "PLACEHOLDER.zarr",  # TODO: replace with the paradigm from PR #84 once it's merged (also pull that)
```

Comment:
reminder to this now that #84 is in

Reply:
i did something based upon #84 but i think there might be some tweaks we want to make to #84 so just to flag we might want to revisit this too